fish-rust/Cargo.toml

```diff
@@ -0,0 +1,47 @@
+[package]
+name = "fish-rust"
+version = "0.1.0"
+edition = "2021"
+
+
+[dependencies]
+widestring-suffix = { path = "./widestring-suffix/" }
+
+autocxx = "0.23.1"
+cxx = "1.0"
+errno = "0.2.8"
+inventory = { version = "0.3.3", optional = true}
+lazy_static = "1.4.0"
```

Comment:
As far as I understand `once_cell` should be used here, it's more flexible and it's even considered for  inclusion in `std`.

Reply:
Not really *considered*, but already there \=)

Most of the useful parts of it are still nightly-locked, so not usable from std yet

Reply:
*Definitely* no nightly features here.

Reply:
Yep, hence "not usable" :)